File: Python Word Examples/ShapesApiDemo/getShapes.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ShapesApi(configuration)

file_name = 'Shape.docx'
extra = {'folder': "input"}
result =api.get_shapes(file_name, **extra)

File: PHP Word Examples/ShapesApiDemo/getShapes.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Configuration;
use \Spire\Cloud\Word\Sdk\Client\Api\ShapesApi;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$fileName = "Shape.docx";
$paragraph_path = 'Section/0/Body/0/Paragraph/0';
$folder ="input";
$storage = null;
$password = null;
$result = $apiInstance->getShapes($fileName,  $folder, $storage, $password,$paragraph_path);